lib/test_track_rails_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module TestTrackRailsClient
-  VERSION = "4.0.0.alpha35" # rubocop:disable Style/MutableConstant
+  VERSION = "4.0.0-rc.1" # rubocop:disable Style/MutableConstant
```

Comment:
Based on other gem conventions I think this should be `4.0.0.rc1`

Reply:
and here i thought i was making the world a better place. oh well, back to dots!

Reply:
Apparently [all you need are one or more letters in the gem version](https://guides.rubygems.org/patterns/#prerelease-gems), but all the examples they include split it with a "." after the minor version, and it seems like a somewhat nuanced parsing problem. So, I figure if Rails does it this way, it's a pretty safe bet.

Reply:
I think it might be for collation order - e.g. you want `4.0.1.rc1` to sort just before `4.0.1`, not before `4.0`, which it might do given the parsing logic in gems.